cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,34 @@
 /// <reference types='cypress' />
 
+
 describe('Book Store app', () => {
-  before(() => {
-    
+
+  it('login', () => {
+    cy.login();
+    cy.visit('/profile');
+    cy.get('#userName-value').should('contain', 'serhio1234');
+    cy.url().should('include', '/profile');
+  });
+
+  it('add book', () => {
+    cy.viewport(750, 600);
+    cy.login();
+    cy.visit('/profile');
+    cy.contains('.btn', 'Book Store').click();
+    cy.get('[placeholder="Type to search"]').click().type('Speaking JavaScript');
+    cy.contains('O\'Reilly Media');
+    cy.contains('.action-buttons', 'Speaking JavaScrip').click();
```

Comment:
There is a typo in the book name, it should be 'Speaking JavaScript' instead of 'Speaking JavaScrip'.

Reply:
Changed